sahi/predict.py

```diff
@@ -332,6 +334,9 @@ def predict(
     Performs prediction for all present images in given folder.
 
     Args:
+        detection_model: sahi.model.DetectionModel
```

Comment:
soyle bir sey daha uygun olur:

`Optionally provide custom DetectionModel to be used for inference. When provided, model_type, model_path, config_path, model_device, model_category_mapping, image_size params will be ignored`